libcontainerd/client_unix.go

```diff
@@ -0,0 +1,147 @@
+// +build solaris linux
```

Comment:
Not sure if these build tags are needed at all. It should work the same without it (and keeping *supporting* freebsd and other unix somehow)